info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java

```diff
@@ -48,10 +54,20 @@
 
 /** A web crawler for compiling candidate-specific news articles information. */
 public class WebCrawler {
-  private static final String CUSTOM_SEARCH_KEY = "";
-  private static final String CUSTOM_SEARCH_ENGINE_ID = "";
-  private static final String TEST_URL =
-      "https://www.cnn.com/2020/06/23/politics/aoc-ny-primary-14th-district/index.html";
+  private static final String CUSTOM_SEARCH_KEY = Config.CUSTOM_SEARCH_KEY;
```

Comment:
You don't need to declare these variables. You can just use Config.CUSTOM_SEARCH_KEY directly.